pkg/installmanager/installmanager.go

```diff
@@ -626,8 +635,10 @@ func (m *InstallManager) provisionCluster() error {
 
 	if err := m.runOpenShiftInstallCommand("create", "cluster"); err != nil {
 		if m.isBootstrapComplete() {
-			m.log.WithError(err).Warn("provisioning cluster failed after completing bootstrapping, waiting longer for install to complete")
-			err = m.runOpenShiftInstallCommand("wait-for", "install-complete")
+			for i := 0; i < m.waitForInstallCompleteExecutions; i++ {
+				m.log.WithError(err).Warn("provisioning cluster failed after completing bootstrapping, waiting longer for install to complete")
```

Comment:
Not sure whether we can have a different message here for every additional wait

Reply:
That's a good idea, would Warnf("[current msg] (%d/%d)", i+1, waitForInstallCompleteExecutions) work?

Reply:
Put the current iteration being performed in a log field.
```suggestion
				m.log.WithField("waitIteration", i).WithError(err).Warn("provisioning cluster failed after completing bootstrapping, waiting longer for install to complete")
```